hack-my-builds.sh

```diff
@@ -38,8 +38,8 @@ _sudo jq --arg bip "$bip" --arg ip "$ip" '
 		| _sudo tee /etc/docker/daemon.json.rawdns \
 		> /dev/null
 _sudo mv /etc/docker/daemon.json.rawdns /etc/docker/daemon.json
-_sudo service docker stop &> /dev/null || :
-_sudo service docker start
+if command -v service; then _sudo service docker stop &> /dev/null; else _sudo systemctl stop docker; fi || :
+if command -v service; then _sudo service docker start; else _sudo systemctl start docker; _sudo systemctl status docker -l --no-pager; fi
```

Comment:
I agree we should support `systemctl` here, but I think we actually ought to prefer it, especially if we detect that systemd is the current init system (https://salsa.debian.org/debian/debhelper/blob/5e2f5a7777cffe4bc7a89fa20dc958954aeda809/autoscripts/postinst-systemd-restart#L2):

```bash
if [ -d /run/systemd/system ] && command -v systemctl; then
	_sudo systemctl stop docker &> /dev/null || :
	_sudo systemctl start docker
	_sudo systemctl status docker --full --no-pager
else
	_sudo service docker stop &> /dev/null || :
	_sudo service docker start
fi
```

Reply:
Done.